### ISSUE TITLE
OL > LI with pagebreak-inside: avoid; breaks number sequence (2, 4, 6, 8...)

### ISSUE THREAD

**Issue by MiChAeLoKGB** [mpdf/mpdf#339]
mPDF has a problem, with rendering ordered lists when list items have `pagebeak-inside` set to `avoid`

Version of mPDF is **6.1**

Here is the code you can use to reproduce this bug:
```
<?php

$body = '<ol><li>1</li><li>2</li><li>3</li><li>4</li><li>5</li><li>6</li><li>7</li><li>8</li></ol>';
$style = 'li { page-break-inside: avoid; }';

$mpdf = new mPDF(); 
$mpdf->WriteHTML($style, 1);
$mpdf->WriteHTML($body,  2);
return $mpdf->Output('test.pdf', 'I');

<?
```

Result:
```
2. 1
4. 2
6. 3
8. 4
10. 5
12. 6
14. 7
16. 8
```

Expected result:
```
1. 1
2. 2
3. 3
4. 4
5. 5
6. 6
7. 7
8. 8
```

**Comment by craigdolphin** [comment]
I have also encountered this issue and can confirm it persists in v 7.0.3.

**Comment by TimStuart** [comment]
We're also encountering this issue.

**Comment by mikestp27** [comment]
Any update on this issue?

**Comment by machal** [comment]
I must confirm this is still a problem. I use mPDF v 8 and after seeting…

```
li {
  page-break-inside: avoid;
}
```

…I got this:

![image](https://user-images.githubusercontent.com/16641/59586328-f5103700-90e2-11e9-8451-9b88def9b4ab.png)

Is there any solution on a way?

**Comment by finwe** [comment]
Should be fixed in v8.0.2

**Comment by machal** [comment]
Thank you! That was fast. :-)
